content/applications/hr/employees/new_employee.rst

```diff
@@ -79,188 +62,271 @@ Resumé tab
-  fields.
-- :guilabel:`Dependant`: If the employee has any dependants, that information is entered here. Type
-  in the number of children the employee has, and check the boxes next to :guilabel:`Disabled
-  Children` and/or :guilabel:`Other Dependent People` if applicable.
-- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
-  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
-  Using the calendar module, select the :guilabel:`Visa Expire Date` to enter the expiration date.
+  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields.
 
 .. image:: new_employee/private-info.png
    :align: center
    :alt: Add the private information to the Private Information tab.
 
-HR settings tab
+HR Settings tab
 ---------------
 
 This tab provides various fields for different information, depending on the country the company is
 located. Different fields are configured for different locations, however some sections appear
 regardless.
 
-- :guilabel:`Status`: If applicable, select a :guilabel:`Related User`, :guilabel:`Job Position`,
-  and :guilabel:`Language` with the drop-down menus. Type in the :guilabel:`Registration Number of
-  the Employee`, and the :guilabel:`NIF Country Code` if available.
-- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
-  Card` information here.
-- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
-  when the employee is working at a work center. This value affects the employee's pay, and may also
-  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
-  amount.
-- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
-  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
-  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
-  :guilabel:`Badge ID` to create a badge ID.
+- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
+  User`, with the drop-down menus.
+
+.. important::
+   Employees do not also need to be users. Employees do **not** count towards billing, while *Users*
+   **do** count towards billing. If the new employee should also be a user, the user must be
+   created. After the :guilabel:`User` is created and saved, the new user will appear in the
+   :guilabel:`Related User` field.
+
+   After the employee is created, create the user. Click the :guilabel:`⚙️ (gear)` icon, then click
+   :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name`,
+   :guilabel:`EmaiL Address`, and then select the :guilabel:`Company` from the drop-down menu. Click
+   :guilabel:`Save` after the information is entered.
```

Comment:
does this automatically link the user and employee? how do I do this manually?

Reply:
Yes and yes, you can in setting. I referenced the doc to explain how, and added the info that it is automatically linked after a new user card is saved.